app.py

```diff
@@ -80,8 +80,8 @@ def shutdown_session(exception):
 from routes import *
 
 def run(self, url=None, debug=False):
-    base_url = url or u'http://localhost:5500/'
-    scheme, netloc, path, parameters, query, fragment = urlparse.urlparse(base_url)
+    base_url = url or 'http://localhost:5500/'
```

Comment:
My only real concern here is stuff like this where we lose the Unicode marker in a string. If we're not moving right to Python 3 I'd like to keep these markers.

It's tough for me to say whether any particular case matters, because we don't have great test coverage of this code base.